### ISSUE TITLE
Insert from Files button not working in TinyMCE editor

### ISSUE THREAD

**Issue by elliot-sawyer** [silverstripe/silverstripe-cms#2915]
### Module version(s) affected

Silverstripe CMS 5.1

### Description

When using a vanilla installation of Silverstripe 5.1, the "Insert From Files" button does not work and causes a console error.

### How to reproduce

1. Login at Silverstripe Demo (demo.silverstripe.org)
2. Create a new Page
3. In the Content field, use "Insert from Files" button
4. Select any existing PDF or upload a new one ([Lorem_ipsum.pdf](https://github.com/silverstripe/silverstripe-cms/files/13943555/Lorem_ipsum.pdf))
5. Enter something for Link Text and description and press Insert Files
6. Observe console error messages 

### Additional Context

```
Uncaught TypeError: e.noticeAdd is not a function
    statusMessage https://demo.silverstripe.org/assets/_tinymce/tinymce-cms-f938cf0f6c.js:62`
```

### Validations

- [X] Check that there isn't already an issue that reports the same bug
- [X] Double check that your reproduction steps work in a fresh installation of [`silverstripe/installer`](https://github.com/silverstripe/silverstripe-installer) (with any code examples you've provided)

## PR
- https://github.com/silverstripe/silverstripe-asset-admin/pull/1436

**Comment by elliot-sawyer** [comment]
Have checked against recipe-cms 5.0.0 as well, appears to be broken there

**Comment by GuySartorelli** [comment]
PR merged. This will be automatically tagged by GitHub actions

**Comment by elliot-sawyer** [comment]
Is the fix for this a patch release on CMS 5.1, or will we need to wait for 5.2?

**Comment by GuySartorelli** [comment]
The fix for this was tagged on `silverstripe/asset-admin` as [2.1.4](https://github.com/silverstripe/silverstripe-asset-admin/releases/tag/2.1.4)
